src/index.html

```diff
@@ -5,9 +5,194 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
     <link rel="stylesheet" href="./styles/main.css">
+    <link rel="icon" type="image/png" href="images/favicon.png">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Miami</h1>
+    <header class="header">
+      <div class="header-wrapper">
```

Comment:
```suggestion
      <div class="header__wrapper">
```
это ведь элемент блока, а не новый блок, как я понял